src/Components/Facility/Consultations/IOBalancePlots.tsx

```diff
@@ -0,0 +1,318 @@
+import moment from "moment";
+import { useCallback, useState } from "react";
+import { useDispatch } from "react-redux";
+import { statusType, useAbortableEffect } from "../../../Common/utils";
+import { dailyRoundsAnalyse } from "../../../Redux/actions";
+
+import ReactECharts from "echarts-for-react";
+
+export const IOBalancePlots = (props: any) => {
+  const { consultationId } = props;
+  const dispatch: any = useDispatch();
+  const [isLoading, setIsLoading] = useState(false);
+  const [offset, setOffset] = useState(0);
+  const [results, setResults] = useState({});
+  const limit = 14;
+
+  const fetchDailyRounds = useCallback(
+    async (status: statusType) => {
+      setIsLoading(true);
+      const res = await dispatch(
+        dailyRoundsAnalyse(
+          {
+            limit,
+            offset,
+            fields: [
+              "infusions",
+              "iv_fluids",
+              "feeds",
+              "total_intake_calculated",
+              "total_output_calculated",
+            ],
+          },
+          { consultationId }
+        )
+      );
+      if (!status.aborted) {
+        if (res && res.data) {
+          setResults(res.data);
+        }
+        setIsLoading(false);
+      }
+    },
+    [consultationId, dispatch, offset]
+  );
+
+  useAbortableEffect((status: statusType) => {
+    fetchDailyRounds(status);
+  }, []);
+
+  const generalOptions = {
+    tooltip: {
+      trigger: "axis",
+    },
+    toolbox: {
+      show: true,
+      feature: {
+        dataZoom: {
+          yAxisIndex: "none",
+        },
+        magicType: { type: ["line", "bar"] },
+        saveAsImage: {},
+      },
+    },
+    xAxis: {
+      type: "category",
+      boundaryGap: false,
+      data: Object.keys(results).map((p: string) => moment(p).format("LLL")),
+    },
+    yAxis: {
+      type: "value",
+    },
+  };
+
+  const infusionsOptions = {
+    ...generalOptions,
+    title: {
+      text: "infusions",
+    },
+    legend: {
+      data: [
+        "Adrenalin",
+        "Dopamine",
+        "Nor-adrenalin",
+        "Vasopressin",
+        "Dobutamine",
+      ],
+    },
```

Comment:
This data will be dynamic. We should convert this to a table. (Means more than the listed medicines can be added to infusions)

The 2 possible plots in IO balance are Total Intake and Total Output.

Reply:
Okay. Meaning I have to remove all plots except `total intake` and `total output` and convert them into table ?